model/src/main/java/com/expediagroup/streamplatform/streamregistry/model/Actor.java

```diff
@@ -12,22 +13,17 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.expediagroup.streamplatform.streamregistry.configuration;
+package com.expediagroup.streamplatform.streamregistry.model;
 
-import java.util.Map;
+import java.util.List;
 
-import javax.validation.Valid;
-
-import lombok.Data;
+import lombok.Builder;
 import lombok.NonNull;
+import lombok.Value;
 
-@Data
-public class SchemaManagerConfig {
-
-    @Valid
-    @NonNull
-    String className;
-
-    Map<String, Object> properties;
-
+@Value
+@Builder
+public class Actor {
```

Comment:
I'm actually for deleting this in the refactor.

Reply:
Agree, These will all changed based on the modelling document.

Reply:
Awesome.